.github/workflows/sql-jdbc-release-workflow.yml

```diff
@@ -37,6 +37,13 @@ jobs:
         run: |
           ./gradlew publishShadowPublicationToInternal-releasesRepository '-Dorg.gradle.jvmargs=--add-modules java.xml.bind' -Dbuild.snapshot=false
 
+      - name: Configure Staging AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
```

Comment:
jdbc needs `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` to build?

Reply:
Currently the build is failing with an `Access denied` error, after a short working session with Sreekar from infra team he suggested using this workaround